### ISSUE TITLE
Add ability to let the facilitator Define voting options

### ISSUE THREAD

**Issue by brucegomes** [Keboo/PointerStar#74]
* Upon creating a room the should be presented with a default option for voting numbers. From here they can accept the default or edit their own options.

Reason: I've worked on teams where stories was pointed 1-5 and 1-21. If we wan this to be usable by any team we need to allow each team to define its voting options.

**Comment by Keboo** [comment]
I agree. The backend currently supports different voting options (defined as an arbitrary array) on the `RoomState` so it should just be a matter of providing the UI and allowing that array to be set.

**Comment by brucegomes** [comment]
@Keboo do you think we should enforce for the facilitator to choose, or just whoever creates the room ?

**Comment by Keboo** [comment]
I would let any facilitator do it, even after the room is created

**Comment by Keboo** [comment]
I would also suggest having some pre-canned sets of options, to let facilitator quickly pick common sets of voting options. I would also suggest persisting the last selection in a cookie the same way we do the user's name. That way when facilitators fire up a new room it defaults to their last selection.
